fdbclient/include/fdbclient/MetaclusterManagement.actor.h

```diff
@@ -1372,19 +1372,26 @@ struct DeleteTenantImpl {
 	MetaclusterOperationContext<DB> ctx;
 
 	// Initialization parameters
-	TenantName tenantName;
+	TenantName tenantName = TenantName();
```

Comment:
This change isn't necessary, it will be default constructed if nothing is specified.